src/overlays/actors/ovl_En_Go2/z_en_go2.c

```diff
@@ -2023,30 +2024,30 @@ s32 EnGo2_DrawRolling(EnGo2* this, PlayState* play) {
 
 s32 EnGo2_OverrideLimbDraw(PlayState* play, s32 limb, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx) {
```

Comment:
Just in case you don't notice, looks like this https://github.com/zeldaret/oot/pull/1458/files#r1045103657 applies here as well.